ruby/ql/src/queries/security/cwe-327/BrokenCryptoAlgorithm.ql

```diff
@@ -23,4 +23,5 @@
   )
   or
   operation.getBlockMode().isWeak() and msgPrefix = "The block mode " + operation.getBlockMode()
-select operation, msgPrefix + " is broken or weak, and should not be used."
+select operation, msgPrefix + " (configured $@) is broken or weak, and should not be used.",
+  operation.getInitialization(), "here"
```

Comment:
We want to avoid using "here" as a link, we prefer it to be more descriptive.  

We have a style guide explaining it: https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#alert-messages.  

Maybe the entire string `"The block mode " + operation.getBlockMode()` can be turned into a link instead?

Reply:
Good idea! Now done.